### PR TITLE
Add DataStructure.schema, DatabaseCategory.schema

### PR DESCRIPTION
**Pull request by tshemsedinov**
Refs: https://github.com/metarhia/metaschema/pull/172

### CODE REVIEW

File: schemas/metaschema/DatabaseCategory.schema [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  Id: { domain: 'Id', required: true },
+  Symbol('allowNotDefinedFields'): false
```
Comment (lundibundi) [inline schemas/metaschema/DatabaseCategory.schema]: Perhaps this was meant to be `Symbol.for`? As `Symbol` will make it impossible to actually get this field IIUC.
Also, I'd suggest to 'namespace' it `@metarhia/allowNotDefinedFields`.

File: schemas/metaschema/DataStructure.schema [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  Symbol.for('AllowNotDefinedFields'): false
```
Comment (belochub) [inline schemas/metaschema/DataStructure.schema]: That's an invalid syntax, you have to enclose the computed property name into the brackets:
```suggestion
  [Symbol.for('AllowNotDefinedFields')]: false
```

File: schemas/metaschema/DatabaseCategory.schema [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  Id: { domain: 'Id', required: true },
+  Symbol.for('AllowNotDefinedFields'): false
```
Comment (belochub) [inline schemas/metaschema/DatabaseCategory.schema]: Ditto.
```suggestion
  [Symbol.for('AllowNotDefinedFields')]: false
```

**Comment by belochub** [comment]
It seems that it was correctly failing on Node.js 6 due to the invalid syntax in the schema definitions, but I am not sure why it hasn't failed on other Node.js versions. This may be caused by a bug in either this package or the `metatests` package.

**Comment by nechaido** [comment]
> @nechaido this field specifies whether not defined fields are allowed or no

@tshemsedinov, I see the purpose of the symbol, yet I can't get how the schemas themselves should be used.

**Comment by nechaido** [comment]
@tshemsedinov ping, see: https://github.com/metarhia/metaschema/pull/174#issuecomment-447256269.

**Comment by nechaido** [comment]
Closed in favor of #221.

**Pull request closed without merging**